test/workflows/components/estimator_runconfig.jsonnet

```diff
@@ -24,7 +24,7 @@ local parts(namespace, name, image) = {
               containers: [
                 {
                   name: "tensorflow",
-                  image: "gcr.io/kubeflow-images-staging/tf-operator-test-server:v20180904-7d89548b",
+                  image: "gcr.io/kubeflow-images-staging/tf-operator-test-server:latest",
```

Comment:
Why latest? Won't that increase the likelihood we suddenly start breaking the tests because someone broke things?

Reply:
Reverted the change.